yjit/src/backend/ir.rs

```diff
@@ -447,9 +447,8 @@ pub enum Insn {
     // Add a label into the IR at the point that this instruction is added.
     Label(Target),
 
-    // Load effective address relative to the current instruction pointer. It
-    // accepts a single signed immediate operand.
-    LeaLabel { target: Target, out: Opnd },
+    /// Get the code address of a jump target
+    LeaJumpTarget { target: Target, out: Opnd },
```

Comment:
Can you explain why the renaming? (not criticizing, genuinely curious)

Reply:
>To derive a full raw pointer from a CodePtr, one needs a base pointer.
Both CodeBlock and VirtualMemory can be used for this purpose. The
base pointer is readily available everywhere, except for in the case of
the jit_return "branch". Generalize lea_label() to lea_jump_target()
in the IR to delay deriving the jit_return address until compile(),
when the base pointer is available.

Basically just so the `jit_return` branch can talk about full address with just access to an assembler (the code block is available only later).